src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -70,21 +70,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_PlatformWithoutConfigurationInference>$(Platform)</_PlatformWithoutConfigurationInference>
   </PropertyGroup>  
   <PropertyGroup Condition=" '$(_PlatformWithoutConfigurationInference)' == 'x64' ">
-    <PlatformTarget>x64</PlatformTarget>
+    <PlatformTarget Condition=" '$(PlatformTarget)' == '' ">x64</PlatformTarget>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_PlatformWithoutConfigurationInference)' == 'x86' ">
-    <PlatformTarget>x86</PlatformTarget>
+    <PlatformTarget Condition=" '$(PlatformTarget)' == '' ">x86</PlatformTarget>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_PlatformWithoutConfigurationInference)' == 'ARM' ">
-    <PlatformTarget>ARM</PlatformTarget>
+    <PlatformTarget Condition=" '$(PlatformTarget)' == '' ">ARM</PlatformTarget>
   </PropertyGroup>
 
   <!-- Default settings for all projects built with this Sdk package -->
   <PropertyGroup>
-    <DebugType>portable</DebugType>
+    <DebugType Condition=" '$(DebugType)' == '' ">portable</DebugType>
 
     <!-- This will turn off the base UWP-specific 'ResolveNuGetPackages' target -->
-    <ResolveNuGetPackages>false</ResolveNuGetPackages>
+    <ResolveNuGetPackages >false</ResolveNuGetPackages>
```

Comment:
Yeah, I was just copying conditions into all the properties, then I thought about it and realized that we don't need to allow changing the NuGet-related properties.